cmd/ignite-spawn/root.go

```diff
@@ -0,0 +1,48 @@
+package main
+
+import (
+	"io"
+
+	"github.com/lithammer/dedent"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"github.com/weaveworks/ignite/cmd/ignite/cmd/cmdutil"
+	"github.com/weaveworks/ignite/pkg/logs"
+)
+
+var logLevel = logrus.InfoLevel
+
+// NewIgniteSpawnCommand returns the root command for ignite-spawn
+func NewIgniteSpawnCommand(in io.Reader, out, err io.Writer) *cobra.Command {
+	root := &cobra.Command{
```

Comment:
I did not expect to see/use cobra in `ignite-spawn` to keep the resource usage of ignite-spawn down to a minimum.
Please use "plain" pflag instead.

Reply:
Refactored in 0568c54.